helm/qontract-reconcile/templates/template.yaml

```diff
@@ -773,6 +902,14 @@ parameters:
   value: ''
 - name: PUSHGATEWAY_SECRET_NAME
   value: push-gateway-auth-url
+### APPSRE-8096: Vector Section Begins
+- name: VECTOR_IMAGE
+  value: timberio/vector
```

Comment:
Please mirror the Vector image to Quay since we can't use Dockerhub images in App Interface.

Reply:
In addition, we need to build this image from ubi9 or eus base images for these changes to be accepted in FedRAMP

Reply:
It could be a great opportunity to build vector with konflux, fwiw

Reply:
I'll look into these as they're both clearly blockers. Not sure about Konflux, but we'll see. I have no preference either way.

Reply:
I'll be using https://catalog.redhat.com/software/containers/openshift-logging/vector-rhel9/6447991f84aeefde2c9f2bcf?container-tabs=overview going forward.

Reply:
LGTM